devtools/cli/src/main/java/io/quarkus/cli/ProjectExtensionsList.java

```diff
@@ -26,6 +26,10 @@
             "--search" }, defaultValue = "*", paramLabel = "PATTERN", order = 3, description = "Search filter on extension list. The format is based on Java Pattern.")
     String searchPattern;
 
+    @CommandLine.Option(names = { "-c", "--cat",
+            "--category" }, defaultValue = "", paramLabel = "CATEGORY_ID", order = 4, description = "Only list extensions from given category.")
+    String category;
```

Comment:
while I know the command is `cat` I would stick with just `-c` and `--category` here.

Reply:
Also: include this option in the dryRun output ~line 67